src/main/kotlin/org/jitsi/nlj/PacketHandler.kt

```diff
@@ -19,6 +19,6 @@ interface PacketHandler {
     /**
      * Process the given packets
      */
-    fun processPackets(pkts: List<PacketInfo>)
+    fun processPacket(packetInfo: PacketInfo)
```

Comment:
We could, but I don't think it's necessary, since iterating is simple enough. In most cases I just had to remove a call to `listOf` or `singletonList`.